sigstore-public-deployment-spec.md

```diff
@@ -0,0 +1,172 @@
+
+In Sigstore’s use of TUF, clients ship with a TUF trust root (which may be rotated over time). From the TUF trust root, clients can derive trust in the public keys of the Sigstore instance (the full details follow the TUF [specification](https://theupdateframework.github.io/specification/latest/) and are out-of-scope for this document).
+
+#### 4.1.1 Artifacts
+
+* Public keys for the certificate transparency logs, e.g. Ctfe.pub  
+* Certificate chains for the CA (Fulcio), e.g. Fulcio\_v1.crt.pem  
+* Public keys for the signature transparency log, e.g. rekor.pub  
+* Public key used to sign the release artifacts: artifact.pub
+
+As keys are rotated over time, older keys have to be provided so clients can verify trust in artifacts produced in the past. Due to this multiple “versions” of the same key material are provided via the TUF repository. By relying on metadata provided, clients can find the correct key material to use for a given artifact.
+
+Transparency logs are subject to sharding, usually once per year to keep the log fairly small. From a verifier, this may be seen as a regular key rotation and the metadata described in the previous paragraph is used to identify the correct key. Clients may also derive the [log ID from the key](https://datatracker.ietf.org/doc/html/rfc6962#section-3.2), and rely on that information when correlating a key to a log.
+
+#### 4.1.2. Roles and Thresholds
+
+* Root  
+  * 5 HSM (Yubikey) keys are held by trusted community individuals across companies, academia, and geolocations.  
+  * Root keys MUST be stored offline, and not used day-to-day.   
+  * Root signing events are expected to occur about every 4-5 months.   
+* Targets  
+  * Secured by the same HSM keys as the root to minimize the number of offline keysets.  
+* Snapshot  
+  * A GCP KMS snapshotting key.  
+  * The snapshot ensures consistency of the metadata files. It has a lifetime of 3 weeks and is re-signed by a GitHub workflow.  
+* Timestamp  
+  * A GCP KMS timestamping key.  
+  * The timestamp indicates the freshness of the metadata files. It has a lifetime of 1 week and is re-signed by two GitHub workflows.
+
+See [sigstore/root-signing repository](https://github.com/sigstore/root-signing) for additional implementation details.
+
+#### 4.1.3 Client
+
+See [Sigstore TUF Client](https://docs.google.com/document/d/1QWBvpwYxOy9njAmd8vpizNQpPti9rd5ugVhji0r3T4c/edit)
+
+## 5. Public Good Instance
```

Comment:
This section feels sparse. I don't have any concrete details of what we should add, and we do need to balance revealing specific configurations that help keep the instance secure. Maybe we need some discussoin on why we made certain decisions?